.github/workflows/typespec-validation-all.yaml

```diff
@@ -41,6 +43,8 @@ jobs:
         shard: [0, 1, 2]
         # total-shards must be an accurate count of the number of shards
         total-shards: [3]
+        # When triggered by schedule, validate both the default branch and typespec-next
+        ref: ${{ fromJSON(github.event_name == 'schedule' && '["default", "typespec-next"]' || '["default"]') }}
```

Comment:
Use `"default"` instead of empty string `""`, since the latter doesn't appear in GitHub Jobs UI for matrix jobs.